extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/UiResource.java

```diff
@@ -78,4 +78,19 @@ public TransferHistoryPage transferHistoryPageEndpoint() {
     public AssetDto getTransferProcessAsset(@PathParam("transferProcessId") String transferProcessId) {
         return transferHistoryPageAssetFetcherService.getAssetForTransferHistoryPage(transferProcessId);
     }
+
+    @GET
+    @Path("pages/policy-page")
+    @Produces(MediaType.APPLICATION_JSON)
+    public PolicyPage policyPage() {
+        return new PolicyPage(policyApiService.getPolicies());
+    }
+
+    @POST
+    @Path("pages/policy-page")
```

Comment:
Nit: Maybe POST pages/policy/page/policies would make more sense, as policies would be the resource, and "pages/policy-page" is just one entry on the resource "pages"